cli-subcommands/create-container.js

```diff
@@ -40,14 +48,27 @@ exports.run = function(probo) {
   var imageConfig = config.images[image];
   if (!imageConfig) return exitWithError('Invalid image ' + image + ' selected.');
   var options = {
-    containerName: probo.config.containerName,
-    docker: config.docker,
-    image: image,
-    build: {},
-    imageConfig: imageConfig,
-    jobConfig: jobConfig,
-    binds: config.binds,
-    attachLogs: true,
+    build: {
```

Comment:
None of these options belong in the build object - they are part of the container config.